src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -1372,11 +1377,8 @@ void WindowScenerySetSelectedTab(const ObjectEntryIndex sceneryGroupIndex)
 // Used after removing objects, in order to avoid crashes.
 void WindowSceneryResetSelectedSceneryItems()
 {
-    auto* w = static_cast<SceneryWindow*>(window_find_by_class(WC_SCENERY));
-    if (w != nullptr)
-    {
-        w->ResetSelectedSceneryItems();
-    }
+    gWindowSceneryTabSelections.clear();
```

Comment:
This will need to be reverted and fixed correctly. You aren't meant to access that global variable when the window is closed.

Reply:
But this list gets constructed when the window is initialized, if necessary. It's a global so that the current selection, per tab, is persistent. It cannot be part of the window class itself, since otherwise the last used scenery per tab is forgotten.